lib/teaspoon/engine.rb

```diff
@@ -98,12 +98,12 @@ def render_exception(_env, exception)
 
 begin
   require 'action_view'
-  if ActionView::VERSION::STRING == '4.2.5'
+  if ActionView::VERSION::STRING == "4.2.5" || ActionView::VERSION::MAJOR >= 5
     require 'action_view/helpers/asset_tag_helper'
     module ActionView::Helpers::AssetTagHelper
       def javascript_include_tag(*sources)
         options = sources.extract_options!.stringify_keys
-        path_options = options.extract!('protocol', 'extname').symbolize_keys
+        path_options = options.extract!('protocol', 'extname', 'host').symbolize_keys
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.